### ISSUE TITLE
`log` integration

### ISSUE THREAD

**Issue by QuietMisdreavus** [QuietMisdreavus/twitter-rs#34]
egg-mode is a pretty huge library, and it would be nice to get some debug info thrown around so i can get useful info in case someone comes in with an issue.

**Comment by passcod** [comment]
This would be especially useful to get the raw json before serde parsing so serde failures due to changing APIs can be diagnosed faster

**Comment by adwhit** [comment]
Yes, I intend to work on this next. PRs always welcome of course!
